packages/salesforcedx-vscode-core/test/jest/conflict/metadataCacheService.test.ts

```diff
@@ -0,0 +1,52 @@
+import { ComponentSet } from '@salesforce/source-deploy-retrieve';
+import { MetadataCacheService } from '../../../src/conflict';
+import { WorkspaceContext } from '../../../src/context';
+import * as sdrUtils from '../../../src/services/sdr/componentSetUtils';
+
+describe('MetadataCacheService', () => {
+  describe('createRetrieveOperation', () => {
+    const dummyComponentSet = new ComponentSet([
+      { fullName: 'Test', type: 'apexclass' },
+      { fullName: 'Test2', type: 'layout' }
+    ]);
+    let retrieveStub: jest.SpyInstance;
+    let getSourceComponentsStub: jest.SpyInstance;
```

Comment:
Weirdly in my IDE I'm seeing 13, 15, and 16 aren't being called, but they clearly are referenced in the beforeEach... 🤔

Reply:
I refactored the test - let me know what you think!